man/systemd.netdev.xml

```diff
@@ -432,6 +432,15 @@
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>EgressQOSMaps=</varname></term>
+        <listitem>
+          <para>The EgressQOSMaps defines a mapping of Linux internal packet priority (SO_PRORITY) to VLAN header
+          PCP field for outgoing frames. A whitespace-separated list unsigned integers in the format from-to.
```

Comment:
"list of"

Would be good to have an example, given it's not a straightforward single-value directive.

Reply:
Yes right will update thanks